geowebcache/core/src/main/java/org/geowebcache/storage/blobstore/file/FileBlobStore.java

```diff
@@ -696,15 +789,108 @@ private Properties getLayerMetadata(final String layerName) {
                         log.warn(e.getMessage(), e);
                     }
                 }
+            } else {
+                log.error(
```

Comment:
Not finding a metadata file is a normal condition, maybe filter parameters were never used before. Why log it at error level?